src/index.ts

```diff
@@ -502,7 +502,14 @@ const ex = createServer((req, res) => {
 
 		return resp
 	}
-	resp.redirect = (location) => { return resp.writeHead(302, { location }) }
+	resp.redirect = (location) => resp.writeHead(302, { location }) 
+	resp.cacheControl = (age) => resp.setHeader('Cache-Control', `public, max-age=${age}`)
```

Comment:
Rename this argument too.
```suggestion
	resp.cacheControl = (maxAge) => resp.setHeader('Cache-Control', `public, max-age=${maxAge}`)
```